packages/batch-submitter/src/batch-submitter/batch-submitter.ts

```diff
@@ -274,6 +278,21 @@ export abstract class BatchSubmitter {
         help: 'Gas used to submit each batch',
         registers: [metrics.registry],
       }),
+      batchesSubmitted: new metrics.client.Counter({
+        name: 'batches_submitted',
+        help: 'Count of batches submitted',
+        registers: [metrics.registry],
+      }),
+      failedSubmissions: new metrics.client.Counter({
```

Comment:
Is this used?

Reply:
updated after the error handling was merged!